shells/android/java/arcs/storage/service/IResultCallback.aidl

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.storage.service;
+
+import arcs.crdt.parcelables.ParcelableCrdtException;
+
+/** Mechanism allowing an asynchronous response from the StorageService. */
+interface IResultCallback {
+    /**
+     * Handles success/failure of an async op.<br><br>
+     *
+     * {@param exception} is nullable.
```

Comment:
Use `@param` like so:

```
@param exception null if {@code success} is true
```

There are some other alternatives you could go with here:
1. Remove the `success` argument, and just go with a nullable exception argument.
2. Provide two methods: `onSuccess` and `onFailure`

Reply:
Thanks for the explanation! I'll come back and fix these javadoc and API issues in a future PR